### DISCUSSION TITLE
Orientation of U1, MCP4651

### DISCUSSION THREAD

**Discussion by electro991** [CrashOverride85/zc95#109] [General]
I started to build the Audio Input board and wanted to share the orientation of U1 in case it can help anyone building it. Hope i got it right but here’s a picture if its correct for others to use. 

![image](https://github.com/user-attachments/assets/476da48b-732c-4765-8d54-c51a2e37d467)

**Comment by CrashOverride85** [comment]
Hi, yes, that's the correct orientation, I'll add a comment to build notes shortly. 
It's using KiCad default footprint for TSSOP-14, where pin 1 is marked by the line on the left of it extending down to the end of pin 1. 
Probably wouldn't have hurt to add a dot next to pin 1 though.
